build-scripts/build_c_tests.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
 
+# Any command line arguments are assumed to be CMake arguments
```

Comment:
might be safer to do everything after `--` but I won't insist on that for this PR

Reply:
Agreed, we won't even need this step once we revamp the build